modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotPartitionsVerifyHandler.java

```diff
@@ -143,7 +144,10 @@ public SnapshotPartitionsVerifyHandler(GridCacheSharedContext<?, ?> cctx) {
             Set<Integer> parts = meta.partitions().get(grpId) == null ? Collections.emptySet() :
                 new HashSet<>(meta.partitions().get(grpId));
 
-            for (File part : cachePartitionFiles(dir, meta.dump() ? DUMP_FILE_EXT : FILE_SUFFIX)) {
+            for (File part : meta.dump()
+                ? cachePartitionFiles(dir, new String[] {DUMP_FILE_EXT, DUMP_FILE_EXT + ZIP_SUFFIX})
```

Comment:
Let's pass `compress` flag to this check and be sure what file extension we expect to find.